charts/alertmanager/templates/statefulset.yaml

```diff
@@ -62,6 +62,10 @@ spec:
             - --webhook-url=http://127.0.0.1:{{ .Values.service.port }}/-/reload
           resources:
             {{- toYaml .Values.configmapReload.resources | nindent 12 }}
+          {{- if .Values.configmapReload.containerPort }}
+          ports:
+            - containerPort: {{ .Values.configmapReload.containerPort }}
```

Comment:
im not totally familiar with the reloader, is there any relation to the webhook-url port?